src/containers/HeatMap/heatMap.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Icon } from 'antd';
+import ReactTooltip from 'react-tooltip';
+import CalenderHeatMap from 'react-calendar-heatmap';
+import moment from 'moment';
+
+import LogoHeader from '../../components/LogoHeader';
+import JournalCard from '../../components/JournalCard';
+import NavigationBar from '../../components/navigationBar';
+import { ReactComponent as NoJournals } from '../assets/images/noJournals.svg';
+import 'react-calendar-heatmap/dist/styles.css';
+import './heatMap.css';
+
+const heatMap = props => {
+  const { data, journals, handleClick } = props;
+
+  const toolTipData = value => {
+    if (value.date) {
+      return {
+        'data-tip': `${value.date.slice(0, 10)} has ${value.count} journals`,
+      };
+    }
+    return {
+      'data-tip': 'No journals yet',
+    };
+  };
+
+  const date = new Date();
+  const firstDay = new Date(date.getFullYear(), date.getMonth(), 1);
+  const lastDay = new Date(date.getFullYear(), date.getMonth() + 1, 0);
```

Comment:
@yosefalnajjarofficial Similar to my comment above, I think these three lines could move outside of the heatMap functional component and just up to the top of the file, since you only need to calculate them once (not once per render).